src/orc.jl

```diff
@@ -418,6 +442,9 @@ end
 Base.unsafe_convert(::Type{API.LLVMOrcIRCompileLayerRef}, il::IRCompileLayer) = il.ref
 
 function emit(il::IRCompileLayer, mr::MaterializationResponsibility, tsm::ThreadSafeModule)
```

Comment:
Hm this function isn't JuliaOJIT specific, but I guess it doesn't hurt emitting this generally?

Reply:
The only possible constructor right now is `IRCompileLayer(jljit::JuliaOJIT)`. I originally was going to add the `jljit` instance to the `IRCompileLayer` to make it properly conditional, but that doesn't seem necessary.